experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1257,61 +1257,121 @@ def test_no_treatment_branches_for_rollout(self):
+            "firefox_min_version": firefox_version,
         }
         serializer = NimbusReviewSerializer(
             experiment, data=data, partial=True, context={"user": self.user}
         )
 
         self.assertTrue(serializer.is_valid())
 
-    def test_rollout_invalid_version_support(self):
-        desktop = NimbusExperiment.Application.DESKTOP
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_101,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+            (
+                NimbusExperiment.Application.FENIX,
+                NimbusExperiment.Version.FIREFOX_101,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_ANDROID,
+                NimbusExperiment.Version.FIREFOX_101,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+            (
+                NimbusExperiment.Application.IOS,
+                NimbusExperiment.Version.FIREFOX_101,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_IOS,
+                NimbusExperiment.Version.FIREFOX_101,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+        ]
+    )
+    def test_rollout_invalid_version_support(
+        self, application, firefox_version, targeting_config
+    ):
         experiment = NimbusExperimentFactory.create_with_lifecycle(
             NimbusExperimentFactory.Lifecycles.CREATED,
-            application=desktop,
+            application=application,
             channel=NimbusExperiment.Channel.NO_CHANNEL,
-            firefox_min_version=NimbusExperiment.Version.FIREFOX_50,
-            feature_configs=[NimbusFeatureConfigFactory(application=desktop)],
+            firefox_min_version=firefox_version,
+            feature_configs=[NimbusFeatureConfigFactory(application=application)],
             is_sticky=True,
             is_rollout=True,
-            targeting_config_slug=NimbusExperiment.TargetingConfig.MAC_ONLY,
+            targeting_config_slug=targeting_config,
         )
         experiment.save()
         for branch in experiment.treatment_branches:
             branch.delete()
 
         data = {
-            "application": NimbusExperiment.Application.DESKTOP,
+            "application": application,
             "is_sticky": "true",
             "is_rollout": "true",
-            "targeting_config_slug": NimbusExperiment.TargetingConfig.MAC_ONLY,
+            "targeting_config_slug": targeting_config,
             "changelog_message": "test changelog message",
-            "channel": "",
-            "firefox_min_version": NimbusExperiment.Version.FIREFOX_50,
+            "channel": NimbusExperiment.Channel.RELEASE,
```

Comment:
why are you asserting release channel?

Reply:
RELEASE is used since for all the Application Configs we have a `Channel.RELEASE` app_id
I should also update channel to `RELEASE` for the experiment

Reply:
Thanks for the feedback, I've updated the channel to release since the other applications require channel.